### PR TITLE
Refactor redirect tests

### LINKED ISSUE #1027: Refactor `test_singular_redirects.py` file
## What issue are you seeing?
A lot of those tests are boilerplate and thusly are far more lines of code than they should be. It'd be great to have a lot of these tests refactored to be something like this: https://github.com/lucyparsons/OpenOversight/blob/5c3b0153a20d9d31692eb200a5a65139c230e360/OpenOversight/tests/routes/test_singular_redirects.py#L630-L646

### PR DESCRIPTION
**Pull request by michplunkett**
## Fixes issue
https://github.com/lucyparsons/OpenOversight/issues/1027

## Description of Changes
Reduce lines in the redirect tests using `@pytest.mark.parametrize`. The coverage stays the same, but the amount of lines use goes dowwwwwwn.

I also moved the `UX-Docs` files.

## Tests and linting
 - [x] This branch is up-to-date with the `develop` branch.
 - [x] `pytest` passes on my local development environment.
 - [x] `pre-commit` passes on my local development environment.

### CODE REVIEW

File: OpenOversight/tests/routes/test_singular_redirects.py [hunk 1]
```diff
@@ -321,214 +358,21 @@ def test_redirect_add_department(client, session):
...
-            follow_redirects=True,
-        )
-        assert resp_redirect.status_code == HTTPStatus.OK
-        assert resp_redirect.request.path == url_for(
-            "main.get_dept_ranks", department_id=AC_DEPT
-        )
-
-
-def test_redirect_get_dept_units(client, session):
-    with current_app.test_request_context():
```
Comment (michplunkett) [inline OpenOversight/tests/routes/test_singular_redirects.py:392]: The `"main.redirect_get_dept_ranks"` function was being tested 3x. 😬

File: OpenOversight/app/filters.py [hunk 2]
```diff
@@ -83,7 +83,7 @@ def display_time(value: datetime) -> str:
 def local_time(value: datetime) -> str:
     """Convert UTC datetime.datetime into a localized time string."""
     if value:
-        return value.astimezone(get_timezone()).strftime("%I:%M %p (%Z)")
+        return value.astimezone(get_timezone()).strftime(f"{OO_TIME_FORMAT} (%Z)")
```
Comment (michplunkett) [inline OpenOversight/app/filters.py:86]: cc: @sea-kelp 

https://github.com/lucyparsons/OpenOversight/pull/1056#pullrequestreview-1621626558

File: OpenOversight/tests/routes/test_singular_redirects.py [hunk 3]
```diff
@@ -20,29 +20,58 @@
...
-        assert resp_redirect.request.path == url_for("main.get_started_labeling")
+        assert resp_redirect.request.path == url_for(route[1])
 
 
-def test_redirect_sort_images(client, session):
+@pytest.mark.parametrize(
+    "route",
+    [
+        ("main.redirect_sort_images", "main.sort_images"),
+        ("main.redirect_edit_department", "main.edit_department"),
```
Comment (sea-kelp) [inline OpenOversight/tests/routes/test_singular_redirects.py]: Could we use 2 names (1 for each value) here? Like `"source_route, target_route"`

Comment (michplunkett) [reply OpenOversight/tests/routes/test_singular_redirects.py]: Done!

**Review by sea-kelp — APPROVED** [review]

**Pull request merged**